src/Avalonia.PlatformSupport/StandardRuntimePlatform.cs

```diff
@@ -169,5 +165,54 @@ void Free(IntPtr ptr, int len)
         IntPtr Alloc(int size) => Marshal.AllocHGlobal(size);
         void Free(IntPtr ptr, int len) => Marshal.FreeHGlobal(ptr);
 #endif
+
+        private static readonly Lazy<RuntimePlatformInfo> Info = new Lazy<RuntimePlatformInfo>(() =>
+        {
+            OperatingSystemType os;
+
+#if NET5_0_OR_GREATER
+            if (OperatingSystem.IsWindows())
+                os = OperatingSystemType.WinNT;
+            else if (OperatingSystem.IsMacOS())
+                os = OperatingSystemType.OSX;
+            else if (OperatingSystem.IsLinux() || OperatingSystem.IsFreeBSD())
+                os = OperatingSystemType.Linux;
+            else if (OperatingSystem.IsAndroid())
+                os = OperatingSystemType.Android;
+            else if (OperatingSystem.IsIOS())
+                os = OperatingSystemType.iOS;
+            else if (OperatingSystem.IsBrowser())
+                os = OperatingSystemType.Browser;
+            else
+                throw new Exception("Unknown OS platform " + RuntimeInformation.OSDescription);
+#else
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                os = OperatingSystemType.OSX;
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+                os = OperatingSystemType.Linux;
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                os = OperatingSystemType.WinNT;
+            else
+                throw new Exception("Unknown OS platform " + RuntimeInformation.OSDescription);
+#endif
+
+            return new RuntimePlatformInfo
+            {
+#if NETCOREAPP
+                IsCoreClr = true,
+#elif NETFRAMEWORK
+                IsDotNetFramework = false,
```

Comment:
IsDotNetFramework = true ?

Reply:
Thanks. Will fix it in another pr later.